package.json

```diff
@@ -30,6 +30,7 @@
     "build:typechain:publicresolver": "typechain --target ethers-v5 --out-dir ethers \"./node_modules/@energyweb/iam-contracts/dist/contracts/PublicResolver.json\"",
     "build:typechain:staking-pool": "typechain --target ethers-v5 --out-dir ethers-staking \"./node_modules/@energyweb/staking-pool/dist/contracts/StakingPool.sol/StakingPool.json\"",
     "prepare": "npm run build && patch-package",
+    "preinstall": "(git apply --ignore-whitespace dist/patches/@walletconnect+ethereum-provider+1.7.1.patch || true) && (git apply --ignore-whitespace dist/patches/@walletconnect+jsonrpc-provider+1.0.0.patch || true)",
```

Comment:
Shouldn't it be in `postinstall`? Because the `node_modules` won't be there in `preinstall`?

Reply:
Yeah, good catch I will fix that